wikibase/1.35/bundle/Dockerfile

```diff
@@ -13,6 +13,8 @@ bash download-extension.sh WikibaseCirrusSearch;\
 bash download-extension.sh UniversalLanguageSelector;\
 bash download-extension.sh cldr;\
 bash download-extension.sh EntitySchema;\
+RUN git clone https://github.com/ProfessionalWiki/WikibaseLocalMedia --depth=1
```

Comment:
Does WikibaseLocalMedia master branch assure compatibility with 1.35 of Mediawiki and Wikibase forever?
Or are we better to pin to a specific commit or other tag etc?

Reply:
If in some years MW 1.35 compat is dropped, could we not just update the branch here? Presumably by that point the Docker images will look differently anyway, and perhaps support a composer package version.

Pinning a specific release will lead to more derp in the short and mid term future.

Reply:
I've created a REL1_35 branch. Hope I don't forgetti to update it when there is a new release. Composer support would still be much better.